### PR TITLE
[ROCKETMQ-200]-Cluster name is always missing when fetch ClusterInfo from name server

### PR DESCRIPTION
**Pull request by Jaskey**
https://issues.apache.org/jira/browse/ROCKETMQ-200

### CODE REVIEW

**Comment by dongeforever** [comment]
LGTM @zhouxinyu @vongosling

File: common/src/main/java/org/apache/rocketmq/common/protocol/route/BrokerData.java [hunk 1]
```diff
@@ -15,9 +15,7 @@
  * limitations under the License.
  */
 
-/**
- * $Id: BrokerData.java 1835 2013-05-16 02:00:50Z vintagewang@apache.org $
- */
+
```
Comment (vongosling) [inline common/src/main/java/org/apache/rocketmq/common/protocol/route/BrokerData.java]: excellent job

File: namesrv/src/main/java/org/apache/rocketmq/namesrv/routeinfo/RouteInfoManager.java [hunk 2]
```diff
@@ -125,11 +125,7 @@ public RegisterBrokerResult registerBroker(
                 BrokerData brokerData = this.brokerAddrTable.get(brokerName);
                 if (null == brokerData) {
                     registerFirst = true;
-                    brokerData = new BrokerData();
-                    brokerData.setBrokerName(brokerName);
-                    HashMap<Long, String> brokerAddrs = new HashMap<Long, String>();
-                    brokerData.setBrokerAddrs(brokerAddrs);
-
+                    brokerData = new BrokerData(clusterName, brokerName, new HashMap<Long, String>());
```
Comment (vongosling) [inline namesrv/src/main/java/org/apache/rocketmq/namesrv/routeinfo/RouteInfoManager.java]: clusterName can be null here ?

**Comment by dongeforever** [comment]
@zhouxinyu @lizhanhui  it is time to merge this PR now, could you please have a double check?

**Comment by zhouxinyu** [comment]
+1, thanks @Jaskey

File: namesrv/src/main/java/org/apache/rocketmq/namesrv/routeinfo/RouteInfoManager.java (hunk 2 above)
Comment (Jaskey) [inline namesrv/src/main/java/org/apache/rocketmq/namesrv/routeinfo/RouteInfoManager.java]: this cluster name is passed from the caller, it should be maintained by the caller.

**Pull request closed without merging**